feature-utils/poly-look/test/react-components/filterChips.test.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import { render, fireEvent } from "@testing-library/react";
+import "@testing-library/jest-dom";
+import FilterChips from "../../src/react-components/filterChips/filterChips.jsx";
+
+const chipsContent = [...Array(3).keys()].map((i) => `chip${i}`);
+const mockedHandleClick = jest.fn();
+const allChipContent = "All";
+// const mockedActiveChips = [chipsContent[1], chipsContent[2]];
+
+const component = (
+  <FilterChips
+    chipsContent={chipsContent}
+    onChipClick={mockedHandleClick}
+    allChip={{ translation: allChipContent }}
+  />
+);
+
+it("Chips present in document", () => {
+  const { getByText } = render(component);
+  for (const chipId of chipsContent) {
+    const chipElement = getByText(chipId);
+    expect(chipElement).toBeInTheDocument();
+  }
+});
+
+it("Checks onChipClick is called", () => {
+  const { getByText } = render(component);
+  for (const chipId of chipsContent) {
+    const chipElement = getByText(chipId);
+    fireEvent.click(chipElement, mockedHandleClick);
+    expect(mockedHandleClick).toHaveBeenCalledWith(chipId, [chipId]);
```

Comment:
Wouldn't clicking on a chip also make it _active_? That seems like the most interesting behaviour to test here - although ensuring that the click handler is invoked, as you already do, also makes sense to me.

Reply:
I'm working on that 😄